website/docs/d/iam_trusted_profiles.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile"
+description: |-
+  Get information about iam_trusted_profile
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile
+
+Provides a read-only data source for iam_trusted_profile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax. For more information about trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-profile
+
+## Example usage
+
+```terraform
+data "ibm_iam_trusted_profile" "iam_trusted_profile" {
+	profile_id = "profile_id"
+}
+```
+
+## Argument reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, Forces new resource, String) ID of the trusted profile to get.
```

Comment:
ID of the trusted profile to get
as
The fetch the trusted profile ID.

Reply:
this is the place where we need to provide the trusted profile id to fetch (get operation)
its an required argument. it makes sense to say 'ID of the trusted profile to get' or 'ID of the trusted profile to fetch' ?

Reply:
ok